build/Makefile

```diff
@@ -57,7 +57,7 @@ KIND_PROFILE ?= agones
 KIND_CONTAINER_NAME=$(KIND_PROFILE)-control-plane
 
 # Game Server image to use while doing end-to-end tests
-GS_TEST_IMAGE ?= gcr.io/agones-images/udp-server:0.21
+GS_TEST_IMAGE ?= gcr.io/agones-dev-286708/game-server:0.1
```

Comment:
Please revert this file in the PR (feel free to keep it locally for testing though).

Reply:
The idea is to use this instead of the udp server, as discussed `[here](https://github.com/googleforgames/agones/pull/1764#discussion_r475817362)`. Do you just prefer that this is changed at a later point?

Reply:
I think @roberthbailey 's point is that it shouldn't be `GS_TEST_IMAGE ?= gcr.io/agones-dev-286708/game-server:0.1` it should be `GS_TEST_IMAGE ?= gcr.io/agones-images/game-server:0.1`

Reply:
It should either be reverted (so that we continue to use the udp-server:0.21 in e2e tests for now) or updated to the agones-images project (if we feel like the new image is ready to go).